<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>Mã cổ phiếu: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="dataTbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>491</width>
     <height>691</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Properties</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="crawlBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nhập dữ liệu tự động</string>
   </property>
  </widget>
  <widget class="QPushButton" name="advisoryBtn">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tư vấn</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>Kết quả tư vấn : </string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>801</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultLb">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>761</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>740</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu kết quả</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ruleTbl">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>801</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Conditions</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addRuleBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm tập luật</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa tập luật</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>Kết quả :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Chỉnh sửa tập luật</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reloadBtn">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tải lại luật</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
